dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_audit_detail.jsp

```diff
@@ -129,8 +128,8 @@
 
     </tr>
     <tr>
-        <th><b><%= LanguageUtil.get(pageContext, "publisher_dialog_force-push") %>: </b></th>
-        <td style="background: white"><%if ( bundle != null ) {%><%=bundle.isForcePush() %><%}%></td>
+        <th><b><%= LanguageUtil.get(pageContext, "contenttypes.content.push_publish.filters") %>: </b></th>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)